nidirect_common/nidirect_common.module

```diff
@@ -187,3 +188,30 @@ function nidirect_common_form_linkit_editor_dialog_form_alter(array &$form, Form
     . '<p>' . t('To reference an e-mail address type the address and select the "E-mail" option from the dropdown list.')
     . '</p>';
 }
+
+/**
+ * Implements hook_views_pre_render().
+ */
+function nidirect_common_views_pre_render(ViewExecutable $view) {
+  if ($view->id() == 'recipes' && $view->current_display == 'page_1') {
```

Comment:
I'd recommend using a display name that denotes the purpose e.g. 'all_recipes'

Reply:
👍